### PR TITLE
Increase test coverage for events category

### PR DESCRIPTION
**Pull request by skumarqa2010**
Test to check page navigation for selected events

### CODE REVIEW

File: cypress/integration/event-signup-flow.spec.js [hunk 1]
```diff
@@ -717,4 +717,94 @@ describe("Verify page load " + new Date(), () => {
 				});
 			});
 	});
+	it('It nativates to selected "Online Events" page', function () {
+		cy.visit("/events/category/online-events", {
```
Comment (ethax-ross) [inline cypress/integration/event-signup-flow.spec.js:721]: Can we extract all the `cy.visit` calls into a function that does the auth so its not repeated every time? not just here but I think there's a bunch of tests that would benefit from it

File: cypress/integration/event-signup-flow.spec.js [hunk 2]
```diff
@@ -717,4 +717,94 @@ describe("Verify page load " + new Date(), () => {
...
+			auth: {
+				username: Cypress.env("HTTPAUTH_USERNAME"),
+				password: Cypress.env("HTTPAUTH_PASSWORD"),
+			},
+		});
+		cy.acceptCookie();
+		cy.get(".events-featured__list__item")
```
Comment (ethax-ross) [inline cypress/integration/event-signup-flow.spec.js:728]: It took me a minute to figure out what was going on here - is there any way to make it clearer? there's so much nesting going on its hard to follow, but it boils down to:

```
event = getFirstEvent();
eventName = getName(event);
navigateTo(event);
pageHeading = getPageHeading();
expect(eventName).to equal(pageHeading);
```

We're also repeating this for each test, can we DRY it up?

Comment (skumarqa2010) [reply cypress/integration/event-signup-flow.spec.js]: Resolved

File: cypress/integration/event-signup-flow.spec.js (hunk 1 above)
Comment (skumarqa2010) [reply cypress/integration/event-signup-flow.spec.js]: Resolved

**Review by ethax-ross — APPROVED** [review]
LGTM
